### PR TITLE
Add EditorConfig

### PR DESCRIPTION
**Pull request by emlun**
To prevent future issues such as the indentation issues in #183.

### CODE REVIEW

**Comment by emlun** [comment]
Ah, this already conflicts with PR #183 starting in commit 89a33323682db0aa752cd44d2a154af0af2ff31d, so let's merge #183 before this and I'll rebase this to fix the conflicts.

File: src/assets/images/cred.png
Comment (gkatrakazas) [reply src/assets/images/cred.png:1]: Take a look to the "files changed"!

Comment (emlun) [reply src/assets/images/cred.png:1]: Whoops, thanks! That was me when i used git's `apply.whitespace = fix` config to remove trailing whitespace by removing and re-adding everything: `git rm - --cached -- . && git add -N . && yes a | git add -up`. The problem was that `git add -p` doesn't catch non-text files, so the images were left deleted. I thought I caught something like that and fixed it, but it seems like I lost the fix somewhere along the way.

**Comment by gkatrakazas** [comment]
Nice, so once we get full ready and merge #183, you can then proceed to close it!

**Comment by gkatrakazas** [comment]
@emlun, the relevant previous PRs have been merged. You have the green light to proceed. When you're ready, ask me for a review just to check before you merge.

**Review by gkatrakazas — APPROVED** [review]

**Pull request merged**
